packages/@react-spectrum/alertbanner/src/AlertBanner.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import AlertMedium from '@spectrum-icons/ui/AlertMedium';
+import {Button, ClearButton} from '@react-spectrum/button';
+import {classNames, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import CrossMedium from '@spectrum-icons/ui/CrossMedium';
+import {DOMProps, DOMRef, StyleProps} from '@react-types/shared';
+import {filterDOMProps} from '@react-aria/utils';
+import InfoMedium from '@spectrum-icons/ui/InfoMedium';
+import React, {ReactNode} from 'react';
+import styles from '@adobe/spectrum-css-temp/components/alertbanner/vars.css';
+import {useProviderProps} from '@react-spectrum/provider';
+
+
+export interface SpectrumAlertBannerProps extends DOMProps, StyleProps {
+  text?: string | ReactNode,
+  variant?: 'neutral' | 'info' | 'negative',
+  actionLabel?: string,
+  onAction?: () => void,
+  isDismissable?: boolean
+}
+
+const ICONS = {
+  info: InfoMedium,
+  negative: AlertMedium
+};
+
+function AlertBanner(props: SpectrumAlertBannerProps, ref: DOMRef<HTMLDivElement>) {
```

Comment:
question to team, is this a landmark?

Reply:
There is a banner landmark: https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/banner.html

> A banner landmark identifies site-oriented content at the beginning of each page within a website. Site-oriented content typically includes things such as the logo or identity of the site sponsor, and site-specific search tool. A banner usually appears at the top of the page and typically spans the full width.

Maybe a question for the accessibility team on whether this qualifies.

Reply:
resolving this one, we're not worrying at this at the moment